funnel/utils/markdown/tabs.py

```diff
@@ -0,0 +1,245 @@
+"""MDIT renderer and helpers for tabs."""
+from __future__ import annotations
```

Comment:
Our convention is to have one blank line after the module docstring. PEP8 doesn't have a rule about this, so linters will accept 0 or 1 — adding a blank line is on us.